NEWS.md

```diff
@@ -1,5 +1,7 @@
 ## rtables 0.5.2.1
  * dev version increment
```

Comment:
package versions are defined in our repo by the first commit to bear that version number in `DESCRIPTION` therefor this isn't really correct. What you have there, if we put it in the news file, will be an entry under `0.5.2.2`, though its an implementation detail so I'm not sure it needs to go in the NEWS file to begin with. That is not to say it wasn't valuable, just not something someone reading a NEWS file is likely to care about

Reply:
it makes sense. `dev version increment` is a placeholder?